### ISSUE TITLE
hyperphosphorylated tau

### ISSUE THREAD

**Issue by sarahstofel** [mgijax/mammalian-phenotype-ontology#3433]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

hyperphosphorylated tau

>2. Synonyms (e.g., absent spleen)

abnormal phosphorylated tau

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

abnormal increased phosphorylation of microtubule associated protein tau in glial cells leading to the formation of neurofibrillary tangles
 
>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID: 28625569

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal brain physiology

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

https://orcid.org/0000-0003-2307-1226

**Comment by sbello** [comment]
Does this mean that there is more phosphorylated tau present or that the process of Tau phosphorylation is enhanced/increased. The definition talks about the process but my initial reading of the label was that you were referring to an increase in the amount of phosphorylated tau. I'm inclined to change this to the level/amount term. Something like "increased phosphorylated tau level" that would be a child of the morphology branch.

**Comment by sbello** [comment]
increased tau phosphorylation

**Comment by sbello** [comment]
also add abnormal and decreased terms

**Comment by sbello** [comment]
There isn't a term in the MP for abnormal brain physiology. 
Based off of this reference https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3696910/ (PMID:23847585) I don't think the phosphorylation occurs just in glial cells "It is also possible that tau propagation may be mediated via glial cells, since cytosolic tau accumulations are observed in neurons surrounded by activated microglia (148) and astrocytes promote tau phosphorylation in neighboring neurons"
I'll use abnormal nervous system physiology as the parent and remove the reference to glial cells in the definition

**Comment by sbello** [comment]
abnormal - MP:0021014
increased - MP:0021015
decreased - MP:0021016

**Comment by sbello** [comment]
definition entered: increase in the process by which a phosphate group is introduced to the tau protein
comment: accumulation of aggregates of hyperphosphorylated tau protein are associated with many neurodegenerative disease including Alzheimer Disease, progressive supranuclear palsy, and some types of frontotemporal dementia
